### PR TITLE
Change default multimedia path

### PR DESCRIPTION
**Pull request by proteusvacuum**
If a form is named, uses form name or module name instead of `module%id`, or `module%id-form%id` for default media filename, else, it defaults to the old name with ids.

http://manage.dimagi.com/default.asp?129713#928696

### CODE REVIEW

File: corehq/apps/app_manager/views.py [hunk 1]
```diff
@@ -916,6 +917,25 @@ def get_details():
         }
 
 
+def _get_default_media_file_name(app, module, form):
+
+    default_file_name = ""
+
+    if module:
+        if trans(module.name, app.langs) == _("Untitled Module"):
```
Comment (czue) [inline corehq/apps/app_manager/views.py]: since this is now being used in logic it might make sense to pull it out into a constant

**Comment by czue** [comment]
(not sure I have enough context on this request/feature to merge)

File: corehq/apps/app_manager/views.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/app_manager/views.py]: +1

**Comment by snopoke** [comment]
It looks like the form / module media name issue was already fixed as part of #4875. I think the other changes are good though (using a constant for the default form name).

**Comment by czue** [comment]
@proteusvacuum what's the status of this?

**Comment by snopoke** [comment]
I think this should be closed

**Comment by proteusvacuum** [comment]
me too.

**Pull request closed without merging**